deploy/10_cloud-ingress-operator-credentials-gcp.CredentialsRequest.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cloud-ingress-operator-credentials-gcp
+  namespace: openshift-cloud-ingress-operator
+spec:
+  secretRef:
+    name: cloud-ingress-operator-credentials-gcp
+    namespace: openshift-cloud-ingress-operator
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/iam.serviceAccountUser
+    - roles/compute.loadBalancerAdmin
+    - roles/compute.securityAdmin
```

Comment:
I don't think we need these
```suggestion
```

Reply:
Ack, they were just blind guesses.  I'll test without them and verify.

Reply:
Looks like you're right.